docs/cli.md

```diff
@@ -28,13 +28,19 @@ It initializes a `pixi.toml` file and also prepares a `.gitignore` to prevent th
 
 - `--channel <CHANNEL> (-c)`: specify a channel that the project uses. Defaults to `conda-forge`. (Allowed to be used more than once)
 - `--platform <PLATFORM> (-p)`: specify a platform that the project supports. (Allowed to be used more than once)
+- `--import <ENV_FILE> (-i)`: Import an existing conda environment file, e.g. `environment.yml`.
+!!! info "Importing an environment.yml"
+    When importing an environment, the `pixi.toml` will be created with the dependencies from the environment file.
+    The `pixi.lock` will be created when you install the environment.
+    We don't support `git+` urls as dependencies for pip packages and for the `defaults` channel we use `main`, `r` and `msys2` as the default channels.
```

Comment:
We could parse the `.condarc` file to figure out the mapping! This is what the `ChannelConfig` struct was meant for. For now though I think this will be fine.

Reply:
Good idea!